src/main/java/graphql/execution/ExecutionStrategy.java

```diff
@@ -620,7 +620,7 @@ protected CompletableFuture<ExecutionResult> completeValueForScalar(ExecutionCon
     protected CompletableFuture<ExecutionResult> completeValueForEnum(ExecutionContext executionContext, ExecutionStrategyParameters parameters, GraphQLEnumType enumType, Object result) {
         Object serialized;
         try {
-            serialized = enumType.serialize(result);
+            serialized = enumType.serialize(result, executionContext.getGraphQLContext(), executionContext.getLocale());
```

Comment:
Boy scout change - the deprecated method should not have been using this mechanism